Demo/src/androidTest/java/com/braintreepayments/demo/test/CardTest.java

```diff
@@ -47,7 +47,7 @@ public void tokenizesACard() {
                 .commit();
 
         onDevice(withText("Card Number")).perform(setText("4111111111111111"));
-        fillInExpiration();
+        onDevice(withText("Expiration Date")).perform(setText("01/27"));
```

Comment:
I opted to hard code a future expiration date rather than change the helper method used to generate a valid expiration year. That helper method is used elsewhere, and I think there is some value in seeing the test inputs when scanning the file. Open to other thoughts on this.

Reply:
I'm indifferent. We could take the middle ground route and create a method like `ExpirationDateHelper.validExpirationText()`. Then you can call
```
onDevice(withText("Expiration Date")).perform(setText(validExpirationText()));
```
after using a static import.

Reply:
Also indifferent on this! Steven's idea is like what we're doing on iOS, FWIW.